dvc/repo/collect.py

```diff
@@ -39,13 +38,6 @@ def _collect_paths(
     for fs_path in fs_paths:
         if recursive and fs.isdir(fs_path):
             target_paths.extend(fs.find(fs_path))
-
-        rel = fs.path.relpath(fs_path)
-        if not fs.exists(fs_path):
```

Comment:
Comment: Here it is internally also hits remote storage. Warnings can be moved to the bottom instead? And we need to make them compact anyways + in the #7692 we will need to handle this better also